packages/components/src/tabs/styles.ts

```diff
@@ -33,25 +33,38 @@ export const TabListWrapper = styled.div`
 		content: '';
 		position: absolute;
 		pointer-events: none;
+		transform-origin: top left;
 
 		// Windows high contrast mode.
 		outline: 2px solid transparent;
 		outline-offset: -1px;
 	}
-	&:not( [aria-orientation='vertical'] )::after {
-		bottom: 0;
-		left: var( --indicator-left );
-		width: var( --indicator-width );
-		height: 0;
-		border-bottom: var( --wp-admin-border-width-focus ) solid
-			${ COLORS.theme.accent };
+	&:not( [aria-orientation='vertical'] ) {
+		&::after {
+			bottom: 0;
+			height: 0;
+			width: 1px;
+			transform: translateX( calc( var( --indicator-left ) * 1px ) )
+				scaleX( var( --indicator-width ) );
+			border-bottom: var( --wp-admin-border-width-focus ) solid
+				${ COLORS.theme.accent };
+		}
+
+		&:dir( rtl )::after {
+			right: 0;
```

Comment:
Instead of setting `right` here as a RTL exception, could we use `inset-inline-start` ?

Reply:
Actually I just realized none are necessary as the pseudo-element naturally falls in the right place by default. So I'll just remove the property 👍